example/image_classification_mobilenet/test/widget_test.dart

```diff
@@ -0,0 +1,30 @@
+// This is a basic Flutter widget test.
```

Comment:
This can be removed. Looking over the example folder, it looks like there's some additional things that don't belong there, probably from trying to make a test project in the examples folder instead of a sub folder (assets, lib, ios, android, build, macos, integration_test, etc.)

Reply:
I am not really sure I know what belongs there and what does not. So I rather left everything in place. But if you give me some pointers, I can remove them.

Reply:
Closing this comment out to unblock the PR since we covered this in the main thread.